spec/lcm/integration/splunk_spec.rb

```diff
@@ -0,0 +1,98 @@
+require_relative 'support/project_helper'
+require_relative 'support/connection_helper'
+require_relative 'support/configuration_helper'
+require_relative 'support/s3_helper'
+require 'securerandom'
+
+def user_in_domain(user_name)
+  domain = @rest_client.domain(LcmConnectionHelper.environment[:prod_organization])
+  domain.find_user_by_login(user_name)
+end
+
+describe GoodData::SplunkLogger do
+  before(:each) do
+    @suffix = ConfigurationHelper.suffix
```

Comment:
could some of the new test structures (project_fixtures class maybe) be used here?